substrate-node/pallets/pallet-tft-bridge/src/tft_bridge.rs

```diff
@@ -0,0 +1,472 @@
+use frame_support::dispatch::DispatchErrorWithPostInfo;
+use frame_support::traits::{Currency, OnUnbalanced, ExistenceRequirement, WithdrawReasons};
+use frame_support::pallet_prelude::DispatchResultWithPostInfo;
+use frame_support::{log, ensure};
```

Comment:
here you could gather what is related to `frame_support` like this:
~~~rust
use frame_support::{
    dispatch::DispatchErrorWithPostInfo,
    traits::{Currency, OnUnbalanced, ExistenceRequirement, WithdrawReasons},
    pallet_prelude::DispatchResultWithPostInfo,
    log, ensure,
};
~~~

Reply:
done, also it would be nice if the project have a `rustfmt.toml` with some defined roles to force consistence format.